src/TinyCiv.Client/Code/ClientManager.cs

```diff
@@ -0,0 +1,28 @@
+﻿using TinyCiv.Server.Client;
+
+namespace TinyCiv.Client.Code
+{
+    public class ClientManager
+    {
+        private static ClientManager instance;
+        public IServerClient Client { get; }
+        public bool requiresUpdate { get; set; }
```

Comment:
```suggestion
        public bool RequiresUpdate { get; set; }
```

C# naming convention

Also, use case of this is unclear, maybe could be removed as it is never used?